src/mumble/AudioOutputSample.h

```diff
@@ -55,14 +55,17 @@ class AudioOutputSample : public AudioOutputUser {
 
 	bool bLoop;
 	bool bEof;
+
+	float fVolume;
```

Comment:
In new code we try to avoid hungarian-style naming of variables. Instead we try to advocate the use of `m_` prefixes for private member variables.
```suggestion
	float m_volume;
```

Reply:
OK, but I recommend writing that down in the ``CODING_GUIDELINES.md``. Especially the fact, that there are multiple different styles currently present in the code base. This is very confusing, especially for someone who tries to mimic the surrounding style as accurately as possible.